git_hub_links.rb

```diff
@@ -33,10 +33,10 @@ def slack_bot_content
     end.map(&:to_s).join("\n")
 
     [
-      '*Weekly dependency update time is here!*',
-      "\nBelow you will find the content for our weekly dependency update spreadsheet",
-      "```#{repo_content}```"
-    ].join(' ')
+      "*Weekly dependency update time is here!*\n",
+      "Below you will find the content for our weekly dependency update spreadsheet\n",
+      "```\n#{repo_content}\n```"
```

Comment:
this doesn't render as "\nxxxxxxxx\n"  ??

Reply:
You can see the output in the screenshots above.

IRB test for you:
```
$ irb
3.2.2 :001 > repo_content = "lots\nof\nlines\nhere"
 => "lots\nof\nlines\nhere" 
3.2.2 :002 > puts "```\n#{repo_content}\n```"
`​`​`​
lots
of
lines
here
`​`​`​
 => nil 
```